alerts/etcdInsufficientMembers.md

```diff
@@ -0,0 +1,81 @@
+# etcdInsufficientMembers
+
+## Meaning
+
+This alert fires when there are fewer instances available than are needed by
+etcd to be healthy.
+
+## Impact
+
+When etcd does not have a majority of instances available the Kubernetes and
+OpenShift APIs will reject read and write requests and operations that preserve
+the health of workloads cannot be performed.
+
+## Diagnosis
+
+This can occur multiple control plane nodes are powered off or are unable to
+connect each other via the network. Check that all control plane nodes are
```

Comment:
can be a followup but I think it's a good user hint to ref firewall explicitly in network partition/issues.

Reply:
Can do here, what kind of wording you were thinking? Do we have any references for this in our docs?

Reply:
Not sure to be honest just in practice a lot of the "hostNetwork is broken" bugs/escalations have been firewall-related. We can work on the wording as a follow-up or even a docs bug if appropriate.